### PR TITLE
Update dawn_qanet_1tpu.json with codeURL

### PR DESCRIPTION
**Pull request by dmrd**

### CODE REVIEW

File: SQuAD/train/dawn_qanet_1tpu.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "model": "QANet",
     "hardware": "1 TPUv2",
     "timestamp": "2018-04-20",
+    "codeURL": "https://github.com/tensorflow/tpu/tree/master/models/experimental/qanet",
```
Comment (codyaustun) [inline SQuAD/train/dawn_qanet_1tpu.json:9]: Please include a commit hash. You can either update the `codeURL` field or add a separate `commitHash` field.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall. 1 small change.

File: SQuAD/train/dawn_qanet_1tpu.json (hunk 1 above)
Comment (dmrd) [reply SQuAD/train/dawn_qanet_1tpu.json:9]: Added commit hash field.

**Comment by dmrd** [comment]
Added the commit hash.

**Review by codyaustun — APPROVED** [review]

**Pull request merged**